wiki/pages/Installation.md

```diff
@@ -26,30 +26,50 @@ source /opt/rez/completion/complete.sh
 
 ## Installation Via Pip
 
-It is possible to install rez with pip, like so:
+It is possible to install rez with pip, like so (from source root directory):
 
 ```
-]$ pip install rez
+]$ pip install --target /opt/rez .
 ```
 
-However, this comes with a caveat - rez command line tools _are not guaranteed
-to work correctly_ once inside a rez environment (ie after using the `rez-env`
-command). The reasons are given in the next section.
+The `.` (dot) can also be replaced with link to the GitHub repository, 
+e.g. choose from:
 
-Pip installation is adequate however, if all you require is the rez API, or you
-don't require its command line tools to be available within a resolved environment.
+  - `git+https://github.com/nerdvegas/rez.git` if you have git installed
+  - `https://github.com/nerdvegas/rez/archive/master.zip`
+  - `https://github.com/nerdvegas/rez/archive/master.tar.gz`
 
-Note that running pip-installed rez command line tools will print a warning like so:
+> [[media/icons/warning.png]] The [PyPi rez](https://pypi.org/project/rez) is 
+> not up to date as of writing. Avoid `pip install rez`
 
-```
-Pip-based rez installation detected. Please be aware that rez command line tools
-are not guaranteed to function correctly in this case. See
-https://github.com/nerdvegas/rez/wiki/Installation#why-not-pip-for-production
-for futher details.
-```
+The environment variables to activate `rez` will be slightly different:
 
+- Add `/opt/rez/bin` to `PATH`
```

Comment:
I don't think we should invite users to add this to their path as it could make them think they can use the rez cli. And here, it only points to the bin folder, which only contains the ``python` binaries...

Reply:
I'll strongly emphasis this and put the warning right after it